src/service/data_products/taxa_count.py

```diff
@@ -232,11 +255,67 @@ async def get_taxa_counts(
     q = await _query(store, collection_id, load_ver, rank)
     for dp_proc in [dp_match, dp_sel]:
         await _add_subset_data_in_place(q, store, collection_id, load_ver, rank, dp_proc)
-        # For now always sort by the std data. See if ppl want sort by match/selection data
-        # before implementing something more sophisticated.
+
+    _sort_taxa_counts(q, sort_priority, [dp_match, dp_sel])
```

Comment:
I'm assuming the next PR is to update _add_subset_data_in_place so it doesn't throw away match & selection data?

Reply:
yea. That's the plan.

Reply:
: